src/core/qgsapplication.h

```diff
@@ -755,6 +763,13 @@ class CORE_EXPORT QgsApplication : public QApplication
      */
     void nullRepresentationChanged();
 
+    /**
+     * requests vor strings of .qgs to be included in ts file
```

Comment:
vor -> for, "requests" -> "Requests"

Actually nicer is

    Emitted when project strings which require translation are being collected for inclusion in a .ts file.

    In order to register translatable strings, connect to this signal and register the strings within the specified \a translationContext.